pkgs/games/multimc/default.nix

```diff
@@ -7,6 +7,11 @@ let
     rev = "5d0ffb50a526173ce58ae57136bf5d79a7e1920d";
     sha256 = "05hnwfb77rmm9ba7n96g4g1sgwqqcmplvbcafsl76yxr6ysgw5jg";
   };
+  quazip_version = "0.7.1";
+  quazip = fetchurl {
```

Comment:
it's even better to resue `quazip` from our pacakges and just reference `quazip.src`